ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -155,6 +155,16 @@ then
 	irlock start -X
 fi
 
+
+# SPL06 sensor external I2C
+if param compare -s SENS_EN_SPL06 1
+then
+	spl06 -X -q start
```

Comment:
You can remove the -q (quiet). If the user sets the parameter and spl06 fails to start we'll want it to be an error.

```suggestion
	spl06 -X start
```